### ISSUE TITLE
Interactive console becomes unresponsive after ";" query

### ISSUE THREAD

**Issue by aidansteele** [turbot/steampipe#700]
**Description**: When I enter a line of *only* a semicolon into the interactive query console, it becomes unresponsive. Ctrl+C and Ctrl+D no longer work. The only option is to Ctrl+Z and then manually SIGKILL the `steampipe` and `postgres` processes.

**Potential cause**:

This code is executed when a query is only the semicolon character:

https://github.com/turbot/steampipe/blob/b7aa42ceb343b037a5cce2057bc048b3eb342f85/interactive/interactive_client.go#L411-L415

It appears to shortcircuit the typical query execution path, so the main REPL hangs here forever:

https://github.com/turbot/steampipe/blob/b7aa42ceb343b037a5cce2057bc048b3eb342f85/interactive/interactive_client.go#L121-L123

**A workaround**: Elsewhere in the file there is a workaround to return control back to the main REPL. Pasting this workaround (`c.resultsStreamer.Done()`) in the first code excerpt does appear to "fix" this bug:

https://github.com/turbot/steampipe/blob/b7aa42ceb343b037a5cce2057bc048b3eb342f85/interactive/interactive_client.go#L153-L155

**Discussion**: To be honest, I'm not 100% sure what the purpose of the "do nothing and restart interactive session" functionality is - so I can't submit a PR that I'm confident is the correct way to fix this. Hope this bug report helps nonetheless.

**Comment by binaek89** [comment]
@aidansteele Thank you for the detailed information on the issue. This will be a great help when debugging the issue

We recently did a major refactor of the `interactive prompt`.

This must have crept in then.

We will take a look into this and publish a fix.

**Comment by kaidaguerre** [comment]
@aidansteele many thanks for digging into this issue. I have released a fix in version Steampipe version 0.7.1

As @binaek mentioned, we have been overhauling the interactive prompt and there is a little more rationalisation still required. 

The fix you identified is the good tactical solution and that is essentially what I have implemented for now. Medium term, I will be looking at simplifying the result streamer to remove underlying issue.
